countries_crawler/src/utils.py

```diff
@@ -0,0 +1,100 @@
+"""Common Functions"""
+
+
+import json
+from bs4 import BeautifulSoup
+import requests
+from constants import BASE_URL, PROVINCE_KEYWORDS
+
+
+def get_response(url):
+    """Get Response from a URL
+
+    This method receive a url of wikipedia and return a response if the status
+    code is 200. Otherwise this method display an error message(invalid url).
+    This will also handle the connection error.
+    """
+
+    response = requests.get(url)
+    if response.ok:
+        return response
+    print(f"\nInavlid URL or Bad Status Code {response.status_code}")
+    return
+
+
+def parse_countries_urls(html):
+    """List of URLs of all countries
+
+    This method receive a response of wikipedia url and return a list of URLs
+    of all countries. These urls are the links of wikipedia containing the
+    detail of individual country.
+    """
+
+    soup = BeautifulSoup(html, "html.parser")
+    country_urls_spans = soup.select("span.flagicon")
+
+    country_urls = [
+        BASE_URL + url_span.parent.a.get("href")
+        for url_span in country_urls_spans
+    ]
```

Comment:
You're allowed to write 80 character per line so let's try to avoid unnecessary line split.